diff -ru wipe-2.2.0.orig/Makefile.in wipe-2.2.0/Makefile.in
--- wipe-2.2.0.orig/Makefile.in	2003-08-31 03:18:03.000000000 +0200
+++ wipe-2.2.0/Makefile.in	2004-01-12 00:58:45.000000000 +0100
@@ -60,12 +60,12 @@
 	$(INSTALL_BIN) -d $(bindir)
 	$(INSTALL_BIN) -s $(BIN_OUT) $(bindir)
 	$(INSTALL) -d $(mandir)/man1
-	$(INSTALL) -o root -m 0644 wipe.1 $(mandir)/man1/
+	$(INSTALL) -m 0644 wipe.1 $(mandir)/man1/
 	rm -rf $(datadir)/doc/wipe*
 	$(INSTALL) -d $(datadir)/doc/wipe
 
 	for file in $(DOCS); do \
-		$(INSTALL) -o root -m 0644 $$file $(datadir)/doc/wipe/; \
+		$(INSTALL) -m 0644 $$file $(datadir)/doc/wipe/; \
 	done
 
 install_home: $(BIN_OUT)
Only in wipe-2.2.0: Makefile.in~
